studio/components/interfaces/Billing/Invoices.tsx

```diff
@@ -117,12 +122,11 @@ const Invoices: FC<Props> = ({ projectRef }) => {
                       </Table.td>
                       <Table.td className="align-right">
                         <div className="flex items-center space-x-2 justify-end">
-                          <a href={x.hosted_invoice_url} target="_blank">
-                            <Button type="outline">View invoice</Button>
```

Comment:
we should not remove View invoice

![Screenshot 2022-03-27 at 1 25 34 PM](https://user-images.githubusercontent.com/689843/160269448-806266fb-f906-4c4f-b082-71ea91fedaeb.png)

![Screenshot 2022-03-27 at 1 24 27 PM](https://user-images.githubusercontent.com/689843/160269426-cc9346f5-a74c-44e8-ad02-51bd05c1fa17.png)